src/test/java/org/fundacionjala/sfdc/stepdefinition/CommonAssertions.java

```diff
@@ -28,9 +32,10 @@ public CommonAssertions(final SFCommonObjects commonPage) {
     @Then("^I verify if \"([^\"]*)\" \"([^\"]*)\" was created/modified on Detail Page$")
     public void iVerifyIfWasCreatedModifiedOnDetailPage(final String nameOfObject,
                                                         final SalesForceEnums.EnumLocator section) {
-        CommonActions.waitTime(2);
+        //CommonWebActions.waitTime(1);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)